### PR TITLE
new updations after the discussion 

### PR DESCRIPTION
**Pull request by suraj-soni-04**

### CODE REVIEW

**Comment by suraj-soni-04** [comment]
Screenshot (168)

Code compiled with no warnings

File: public/favicon.ico
Comment (vinayaksutar) [inline public/favicon.ico:1]: Remove if not required

File: src/reportWebVitals.ts
Comment (vinayaksutar) [inline src/reportWebVitals.ts:1]: remove if not required

File: src/logo.svg
Comment (vinayaksutar) [inline src/logo.svg:1]: remove if not required

File: public/robots.txt
Comment (vinayaksutar) [inline public/robots.txt:1]: remove if not required

**Review by vinayaksutar — CHANGES_REQUESTED** [review]
Code cleanup

**Comment by suraj-soni-04** [comment]
Ok, doing the changes

File: public/favicon.ico
Comment (suraj-soni-04) [reply public/favicon.ico:1]: removed

File: src/reportWebVitals.ts
Comment (suraj-soni-04) [reply src/reportWebVitals.ts:1]: it was required

File: src/logo.svg
Comment (suraj-soni-04) [reply src/logo.svg:1]: removed

File: public/robots.txt
Comment (suraj-soni-04) [reply public/robots.txt:1]: removed

**Comment by suraj-soni-04** [comment]
suggested clean is done

**Review by vinayaksutar — APPROVED** [review]

**Pull request merged**